src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -100,6 +100,15 @@ class LocalBackend extends Backend with SharedFileSystem {
     }
   } map CompletedExecutionHandle
 
+  def useCachedCall(cachedBackendCall: BackendCall, backendCall: BackendCall)(implicit ec: ExecutionContext): Future[ExecutionHandle] = Future {
+    Try(cachedBackendCall.callRootPath.copyTo(backendCall.callRootPath)) match {
+      case Success(_) =>
```

Comment:
actually this is incomplete because of the `.get`... I have to match the output of postProcess and handle failure properly